.circleci/config.yml

```diff
@@ -84,6 +84,44 @@ jobs:
           failure_message: Nightly << parameters.stack >> auth tests failed!
           webhook: $SLACK_WEBHOOK
 
+  audit:
+    working_directory: ~/repo
+    docker:
+      - image: cimg/base:stable-20.04
+        auth:
+          username: dockstoretestuser
+          password: $DOCKERHUB_PASSWORD
+    steps:
+      - browser-tools/install-browser-tools
+      - checkout
+      - run:
+          name: Checkout merge commit (PRs only)
+          command: |
+            if [[ -n "${CIRCLE_PULL_REQUEST}" ]]
+            then
+              git fetch origin +refs/pull/${CIRCLE_PULL_REQUEST##*/}/merge:
+              git checkout -qf FETCH_HEAD
+            fi
+      - install_container_dependencies
+      - run:
+          name: Run npm audit
+          #Only run 'npm ci' if node_modules was not restored from cache
```

Comment:
Confused by this comment; you don't run `npm ci` here ever, but comment implies you might.

And why don't you run `npm ci`? Won't `npm i` skip package-lock.json, which has the actual dependencies we are using?

Not sure on my questions, just asking.

Reply:
I left that comment in by accident. 

`npm i` doesn't skip the package-lock.json and may even write to that file. You're supposed to use this when you install a new package in the package.json. But you were right, I should have been using `npm ci` anyway. I wasn't before because I added a dependency in order to make the tests fail and `npm ci` will fail when there is a mismatch between the package.json and package-lock.json...and there was a mismatch because I didn't run `npm i`.